src/main/java/us/kbase/auth2/providers/OrcIDIdentityProviderFactory.java

```diff
@@ -228,6 +233,58 @@ private OrcIDAccessTokenResponse(
 				this.accessToken = accessToken.trim();
 				this.fullName = fullName == null ? null : fullName.trim();
 				this.orcID = orcID.trim();
+				this.idToken = idToken == null ? null : idToken.trim();
+				this.mfaAuthenticated = parseAmrClaim(this.idToken);
+			}
+			
+			/**
+			 * Parses the Authentication Method Reference (AMR) claim from an OpenID Connect ID token
+			 * to determine if multi-factor authentication was used.
+			 * 
+			 * @param idToken the JWT ID token from ORCID
+			 * @return true if MFA was used, false if password only, null if unknown or parsing failed
+			 */
+			private Boolean parseAmrClaim(final String idToken) {
+				if (idToken == null || idToken.trim().isEmpty()) {
+					return null;
+				}
+				
+				try {
+					// JWT format: header.payload.signature
+					final String[] parts = idToken.split("\\.");
+					if (parts.length != 3) {
+						// Invalid JWT format
+						return null;
+					}
+					
+					// Decode the payload (second part) - URL-safe base64
+					final String payload = new String(Base64.getUrlDecoder().decode(parts[1]));
+					
+					// Parse JSON payload to extract claims
+					@SuppressWarnings("unchecked")
+					final Map<String, Object> claims = MAPPER.readValue(payload, Map.class);
+					
+					final Object amrClaim = claims.get("amr");
+					if (amrClaim instanceof List) {
+						// OpenID Connect spec: AMR should be an array of strings
+						@SuppressWarnings("unchecked")
+						final List<String> amrList = (List<String>) amrClaim;
+						return amrList.contains("mfa");
+					} else if (amrClaim instanceof String) {
+						// ORCID may return single string - handle as fallback
+						return "mfa".equals(amrClaim);
+					}
+					
+					// AMR claim present but in unexpected format
+					return null;
+					
+				} catch (IllegalArgumentException e) {
+					// Base64 decoding failed - invalid JWT
+					return null;
+				} catch (IOException e) {
+					// JSON parsing failed - malformed payload
+					return null;
```

Comment:
Could optionally throw an error in these cases, but we would need to handle the error. Perhaps we just log this case? If the JWT is correct, we shouldn't see these. Thoughts?